packages/babel-plugin-transform-nullish-coalescing-operator/README.md

```diff
@@ -0,0 +1,50 @@
+# @babel/plugin-transform-nullish-coalescing-operator
+
+> Replace `??` with an inline helper.
+
+## Example
+
+**In**
+
+```javascript
+var foo = object.foo ?? "default";
+```
+
+**Out**
+
+```javascript
+var _ref;
+var foo = (_ref = object.foo, _ref != null ? _ref : "default");
```

Comment:
to clarify; i think that using `!= null` by default should be a blocker, since it's not in compliance with the spec